app/models/comment.rb

```diff
@@ -3,7 +3,7 @@ class Comment < ActiveRecord::Base
   belongs_to :user
   validates :body, :presence => :true
 
-  default_scope order: 'comments.created_at DESC'
+  default_scope { order('comments.created_at DESC')}
```

Comment:
Support for default_scope without block was removed from rails 4.1. You should look [here](http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#changes-on-default-scopes#changes-on-default-scopes) and [here](http://stackoverflow.com/questions/24163571/default-scope-activerecord-4-1-block-usage-for-ordering#answer-24164710) 
Although, doc of default_scope still says we can use ones without block. But error in log clearly said that default_scope without blocks is no longer supported. I think that stackoverflow question explains it a bit.